gis/mapserver/Portfile

```
@@ -1,122 +1,170 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake   1.1
 
 name                mapserver
-version             6.2.4
-revision            9
+version             8.0.1
+revision            0
 maintainers         hbaspecto.com:jea openmaintainer
 categories          gis
 license             permissive
-platforms           darwin
 
 description         mapserver
-long_description    MapServer is an Open Source development environment for \
-                    building spatially-enabled internet applications.
+long_description    MapServer is an Open Source platform for publishing spatial \
+                    data and interactive mapping applications to the web.
 
-homepage            http://mapserver.org
-master_sites        http://download.osgeo.org/mapserver
+homepage            https://mapserver.org
+master_sites        https://download.osgeo.org/mapserver
 
-checksums           rmd160  690374d43a1b733c3b7acd73c37099d95554c599 \
-                    sha256  237d83f84042fef8a63c590256aaa3bc1400668a470b52252fae197d9689e41c
+checksums           rmd160  e1601b6d872c9f2cc757ddeee6dad13523809d2c \
+                    sha256  79d23595ef95d61d3d728ae5e60850a3dbfbf58a46953b4fdc8e6e0ffe5748ba \
+                    size    2950164
 
 depends_build       port:pkgconfig
```

CI fails:
```
sh: /opt/local/bin/cmake: No such file or directory
```
because the port overwrites the cmake portgroup's dependencies. To fix it, append instead of overwriting:
```suggestion
depends_build-append \
                    port:pkgconfig
```